src/main/webapp/one-off/databaseInit.java

```diff
@@ -187,15 +187,16 @@ private Entity buildRecipeEntity(
       "Lorem quam dolor dapibus ante, sit amet pellentesque turpis lacus eu ipsum. Duis quis mi ut tortor interdum efficitur quis at mi. Pellentesque quis mauris vel ligula commodo scelerisque. In vulputate quam nisl, vel sagittis ipsum molestie quis. Suspendisse quis ipsum a sem aliquam euismod mattis sed metus.";
     Random rd = new Random();
     Double number = rd.nextDouble();
+    Double prep_time = 30.00;
```

Comment:
Why is this a double if in the backend it's an integer?

Reply:
I got some errors when requesting the value from the entity, I manage to solve them by using Double. I'll change the value to int.